core/model/integration.py

```diff
@@ -112,6 +131,10 @@ class IntegrationLibraryConfiguration(Base):
     It stores the configuration settings for each external integration in
     a single json row in the database. These settings are then serialized
     using Pydantic to a python object.
+
+    This is a many-to-many relationship between IntegrationConfiguration and
+    Library. Implementing the Association Object pattern:
+    https://docs.sqlalchemy.org/en/14/orm/basic_relationships.html#association-object
     """
```

Comment:
This has always been the case, I just wanted to clarify and formalize how this table was being used.